### DISCUSSION TITLE
Please make also bypass lightspeed filter agent extension, and work if chrome web store is blocked by google

### DISCUSSION THREAD

**Discussion by Kenfin777** [3kh0/ext-remover#922] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
please

**Comment by PenguinLord561** [comment]
did you test all of the ones here? https://github.com/3kh0/ext-remover

**Reply by Kenfin777** [reply]
Hey I think my school paid for "Lightspeed Filter Agent", which is why nobody has 
a way to bypass it.

**Reply by AshtonDavies** [reply]
Lightspeed Filter Agent is so bypassable.

**Reply by Blobby-Boi** [reply, marked as answer]
Use LTMEAT Flood or LTMEAT Print or Dextensify (I have Lightspeed too)

**Reply by v4pl** [reply]
Skiovox on top

**Reply by AshtonDavies** [reply]
> Skiovox on top

Except that it's a browser in a kiosk profile, meaning you don't have files, apps, and sync. It's just the Chrome browser.

**Reply by 1coolmatt8083** [reply]
hey guys does any one have a way to play roblox with the light speed agent instaled

**Reply by myschoolisass** [reply]
https://now.gg/apps/roblox-corporation/5349/roblox.html
._.
"oh no it doesn't work" **use the options listed by blobby boi**

**Reply by Voidless7125** [reply]
> Hey I think my school paid for "Lightspeed Filter Agent", which is why nobody has a way to bypass it.

I have found a Bypass for a website you have JS access to.
```
 function timeout(ms) {
        return new Promise(resolve => setTimeout(resolve, ms));
    }

    async function Never() {
        await timeout(75);
        console.clear();
        console.log("=====");

        // Functions
        var getLoaderPolicy = function () { }    // no-op function;
        var loadES6 = function () { }    // no-op function;
        var isYoutube = function () { }    // no-op function;
        var checkCurrentSite = function () { }    // no-op function;
        var getHardBlockPolicy = function () { }    // no-op function;
        var hardBlock = function () { }    // no-op function;
        var stopVideo = function () { }    // no-op function;
        var updateLocation = function () { }    // no-op function;

        // Variables
        var hardBlockPolicy = null;
        var prevURL = null;

        console.log("=====");

        // re-assign
        window.isYoutube = function () { }    // no-op function
        window.loadES6 = function () { }    // no-op function
        window.checkCurrentSite = function () { }    // no-op function
        window.getHardBlockPolicy = function () { }    // no-op function
        window.hardBlock = function () { }    // no-op function
        window.stopVideo = function () { }    // no-op function
        window.updateLocation = function () { }    // no-op function
        window.initFlagScanning = function () { }    // no-op function
        window.getLoaderPolicy = function () { }    // no-op function
        window.loaderPolicy = function () { }    // no-op function

        console.log("Just incase, Functions deleted again. This is not needed, but we Bypassed the filter agent. =)");

        console.log("=====");
    }
Never()
```
Add this to your dom loader check, and adjust your delay as necessary.  (Faster computers need a faster timeout, but slow computers need a longer timeout, and if the timing is wrong, you might miss deleting the vars before or after.)

My website (cat as unknown)
https://ranchodvt.github.io/Comp-V3/index.html

**Comment by KandyKaneMarvel** [comment]
try downgrading and using something like skiovox i guess, seems like the best option. Or, you know, just use game websites.

**Comment by SCPlayz7000** [comment]
get off the school laptop lil bro :)

**Comment by SCPlayz7000** [comment]
do your work don't play games
dislike=your a kid

**Reply by S-PScripts** [reply]
necropost = _you're_ a kid

**Reply by SCPlayz7000** [reply]
self ownage+L ratio+british discord unc+illegal exploitation and destruction of school property+violation of github tos

**Comment by SCPlayz7000** [comment]
Yall just can't help yourselves, can you? This is destruction of school
property.[image: Screenshot 2024-12-05 094116.png]
[image: Screenshot 2024-12-05 094603.png]
[image: Screenshot 2024-12-05 094609.png]
[image: Screenshot 2024-12-05 094618.png]
[image: Screenshot 2024-12-05 094753.png]
[image: Screenshot 2024-12-05 094821.png]
[image: Screenshot 2024-12-05 094844.png]
[image: Screenshot 2024-12-05 094855.png]
[image: Screenshot 2024-12-05 094950.png]
[image: Screenshot 2024-12-05 095101.png]
[image: Screenshot 2024-12-05 095144.png]
[image: Screenshot 2024-12-05 095153.png]
[image: Screenshot 2024-12-05 095213.png]
[image: Screenshot 2024-12-05 095225.png]
[image: Screenshot 2024-12-05 095238.png]
[image: Screenshot 2024-12-05 095249.png]
[image: Screenshot 2024-12-05 095258.png]
[image: Screenshot 2024-12-05 095324.png]
[image: Screenshot 2024-12-05 095334.png]
[image: Screenshot 2024-12-05 095343.png]
[image: Screenshot 2024-12-05 095352.png]
[image: Screenshot 2024-12-05 095403.png]
[image: Screenshot 2024-12-05 095411.png]
[image: Screenshot 2024-12-05 095436.png]
[image: Screenshot 2024-12-05 095449.png]
[image: Screenshot 2024-12-05 095459.png]
[image: Screenshot 2024-12-05 095515.png]



<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Reply by AshtonDavies** [reply]
you don't know how to use github discussions do you?
